src/commands/oas2tson.ts

```diff
@@ -1,6 +1,6 @@
 import $RefParser, { ParserOptions } from '@apidevtools/json-schema-ref-parser'
 import { Command } from 'commander'
-import filenamify from 'filenamify'
+// import filenamify from 'filenamify'
```

Comment:
what's not used must be removed

Reply:
I missed that in cleanup. Thanks.

https://github.com/nearform/openapi-transformer-toolkit/pull/318/commits/941d116acc8c2c1f529e7d7e9e0bce899bd47e14